docs/concepts/apiserver_admission_plugins.md

```diff
@@ -54,6 +54,12 @@ This admission controller reacts on `CREATE` and `UPDATE` operations for `Backup
 For all the various extension types in the specifications of these objects, it validates whether there exists a `ControllerRegistration` in the system that is primarily responsible for the stated extension type(s).
 This prevents misconfigurations that would otherwise allow users to create such resources with extension types that don't exist in the cluster, effectively leading to failing reconciliation loops.
 
+## `ExtensionLabels`
```

Comment:
We don't have any docs about extension validators in the garden cluster in https://github.com/gardener/gardener/tree/master/docs/extensions yet.
I think, this would be a good time to start a doc for that. It could mention, that extensions are expected to validate their shoots, cloudprofiles etc., some best practices (e.g., only validate on spec changes), and that they should add object selectors to filter for objects they are responsible for.

Reply:
Is this what you have in mind? gardener@ed4370d

Reply:
Can you PTAL? @rfranzke